lib/spack/spack/cmd/install.py

```diff
@@ -115,7 +116,13 @@ def setup_parser(subparser):
         action="store_true",
         help="if a partial install is detected, don't delete prior state",
     )
-
+    subparser.add_argument(
+        "--cmake-external-stage",
```

Comment:
Can you do without adding yet another flag to `install` please.

Reply:
Not a problem. I was err-ing on the side of usability but if the install CLI is already bloated, then yes, I don't see a reason this _needs_ to be there